opensrp-eusm/src/main/java/org/smartregister/eusm/util/AppUtils.java

```diff
@@ -249,4 +251,27 @@ public static void updateLocationCoordinatesFromForm(StructureDetail structureDe
             structureRepository.addOrUpdate(location);
         }
     }
+
+    public static ArrayList<String> getRegionsForDistricts(JSONArray locationHierarchy, Set<String> districts, boolean hideDistricts) throws Exception {
+        ArrayList<String> opRegions = new ArrayList<>();
+        JSONArray regions = locationHierarchy.getJSONObject(0).getJSONArray(AppConstants.JsonForm.NODES);
+        for (int i = 0; i < regions.length(); i++) {
+            JSONObject location = regions.getJSONObject(i);
+            String locName = location.getString(TaskingConstants.CONFIGURATION.KEY);
+            if (location.has(AppConstants.JsonForm.NODES)) {
+                JSONArray childNodes = location.getJSONArray(AppConstants.JsonForm.NODES);
```

Comment:
Check for null on `childNodes`.

Reply:
`childNodes` will not be null as we already have a check in place at line `261` which will make sure that the current location has nodes